blazor/datagrid/grouping.md

```diff
@@ -296,4 +296,60 @@ The following sample code demonstrates the above,
 The following screenshot represents a DataGrid with customized group caption,
 ![Blazor DataGrid with Custom Group Caption](./images/blazor-datagrid-custom-group-caption.png)
 
+## Group column programmatically
+
+The columns in a Grid can be dynamically grouped and ungrouped using the [GroupColumnAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_GroupColumnAsync_System_String_) and [UngroupColumnAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_UngroupColumnAsync_System_String_) methods.
+
+As demonstrated in the following sample, the Customer Name column is grouped and ungrouped dynamically through the Group Column and UnGroup Columns buttons utilizing by these methods.
```

Comment:
As demonstrated in the following sample, the Customer Name column is grouped and ungrouped dynamically through the Group Column and UnGroup Columns buttons utilizing these methods.